src/main/resources/templates/cdn/synapse-data-cdn.json.vtp

```diff
@@ -0,0 +1,116 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Provision a CloudFront distribution for synapse data",
+  "Resources": {
+    "CloudFront": {
+      "Type": "AWS::CloudFront::Distribution",
+      "DependsOn": [
+        "CloudFrontPublicKeyGroup",
+        "OriginAccessControl"
+      ],
+      "Properties": {
+        "DistributionConfig": {
+          "Comment": "CloudFront distribution for ${stack}data.sagebase.org",
+          "Origins": [
+            {
+              "DomainName": "${stack}data.sagebase.org.s3.us-east-1.amazonaws.com",
+              "Id": "${stack}data.sagebase.org",
+              "OriginAccessControlId": {
+                "Ref": "OriginAccessControl"
+              },
+              "S3OriginConfig": {
+                "OriginAccessIdentity": ""
+              }
+            }
+          ],
+          "DefaultCacheBehavior": {
+            "AllowedMethods": [
+              "GET",
+              "HEAD"
+            ],
+            "CachePolicyId": "4135ea2d-6df8-44a3-9df3-4b5a84be39ad",
+            "OriginRequestPolicyId": "b689b0a8-53d0-40ab-baf2-68738e2966ac",
+            "TargetOriginId": "${stack}data.sagebase.org",
+            "TrustedKeyGroups": [
+              {
+                "Ref": "CloudFrontPublicKeyGroup"
+              }
+            ],
+            "ViewerProtocolPolicy": "https-only"
+          },
+          "Logging": {
+            "Bucket": "${stack}.log.sagebase.org.s3.us-east-1.amazonaws.com",
+            "IncludeCookies": true,
+            "Prefix": "cloudFrontDataRequests"
```

Comment:
we need to look if enabling Manage additional metrics adds extra cost or its free.